www/src/components/diagram.js

```diff
@@ -316,8 +323,31 @@ const Diagram = () => (
       >
         <ItemTitle>Static Web Host</ItemTitle>
         <ItemDescription>
-          Amazon S3, Netlify, GitHub Pages, Surge.sh, Aerobatic, Now.sh, & many
-          more
+          <StaticQuery
```

Comment:
I usually hoist these to the top child element (i.e. wrapping the top `div` in this example).

More of a personal preference, so I'm curious what others think! What do you think @amberleyromo @pieh

Reply:
Done! ✅ Thanks for the suggestion 😄